scripts/py_matter_idl/matter_idl/test_data_model_xml.py

```diff
@@ -72,6 +72,46 @@ def testBasicInput(self):
 
         self.assertEqual(xml_idl, expected_idl)
 
+    def testEnumRange(self):
+        # Check heuristic for enum ranges
```

Comment:
Please add cases for 8 bits and 16 bits, and include a case where there is a single bit and it's above 32 bits.

Reply:
Done. Based on offline discussion:
   - removed enum32 (not a thing)
   - added bitmap64 and unit test